app/src/main/java/org/wordpress/pioupiou/login/LoginActivity.java

```diff
@@ -140,6 +143,17 @@ private void attemptLogin() {
             Timber.i("Start login process using XMLRPC API on: " + mUrl);
             // Self Hosted login
             // TODO: insert cool stuff here
+
+            // TODO: we might want to check that all needed methods are exposed in the XML RPC interface
```

Comment:
The discovery process actually already handles this - it will eventually send a `system.listMethods` request to the site and check that the response contains all the methods we expect [here](https://github.com/wordpress-mobile/WordPress-FluxC-Android/blob/ed116e5591c1ecb629e737ead31d53f9f8573233/fluxc/src/main/java/org/wordpress/android/fluxc/network/discovery/DiscoveryUtils.java#L66-L94).

If there are missing methods, the discovery process will [surface a `MISSING_XMLRPC_METHOD` error](https://github.com/wordpress-mobile/WordPress-FluxC-Android/blob/ed116e5591c1ecb629e737ead31d53f9f8573233/fluxc/src/main/java/org/wordpress/android/fluxc/network/discovery/SelfHostedEndpointFinder.java#L386) through the `OnDiscoveryResponse` event.